app/models/manual.rb

```diff
@@ -1,6 +1,8 @@
 class Manual
   include ActiveModel::Model
   include ActiveModel::Validations
+  #publishing-api will return per_page default of 50 items. ALL variable is set to high number to return all manuals until pagination is properly implemented, else user would only see 50 manuals.
+  ALL = 10000
```

Comment:
I feel uneasy about using a constant, but I'm not sure what is best.
Secondly, I don't think ALL is a good name, as it is semantically incorrect.
 I might use something like this
```
def self.max_numbers_of_manuals
  10000
end
```

disclaimer, I'm not sure what is best, but I want to call this out for more opinions.